oak/server/module_invocation.cc

```diff
@@ -77,11 +81,19 @@ void ModuleInvocation::ProcessRequest(bool ok) {
   std::string encap_req;
   grpc_request.SerializeToString(&encap_req);
   // TODO: figure out a way to avoid the extra copy (into then out of std::string)
-  std::unique_ptr<Message> rsp_msg = absl::make_unique<Message>();
-  rsp_msg->data.insert(rsp_msg->data.end(), encap_req.begin(), encap_req.end());
+  std::unique_ptr<Message> req_msg = absl::make_unique<Message>();
+  req_msg->data.insert(req_msg->data.end(), encap_req.begin(), encap_req.end());
+  {
```

Comment:
Why the block?

Reply:
I like to use blocks to minimize scope of auxiliary variables, such as `range` in this context.